### PR TITLE
LPD-18034 Setting cache-control header to private only when there's no info about languageId

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hey,

This pull request is the result from the conversation [here](https://liferay.atlassian.net/browse/LPP-52157) between @izaera , @holatuwol and @bryceosterhaus .

As you can see by [this comment](https://liferay.atlassian.net/browse/LPP-52157?focusedCommentId=2550255) the idea is to only set `cache-control` header to `private` when there's no information about `languageId` in URL.

Thanks.

Regards.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/servlet/filters/language/LanguageFilter.java [hunk 1]
```diff
@@ -75,8 +76,17 @@ protected void processFilter(
 			HttpServletResponse httpServletResponse, FilterChain filterChain)
 		throws Exception {
 
-		httpServletResponse.setHeader(
-			HttpHeaders.CACHE_CONTROL, "private, no-cache");
+		String queryString = httpServletRequest.getQueryString();
```
Comment (izaera) [inline portal-impl/src/com/liferay/portal/servlet/filters/language/LanguageFilter.java:79]: :thinking: Why don't you use [ServletRequest.getParameter](https://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html#getParameter(java.lang.String))?

**Comment by antonio-ortega** [comment]
Closing this since the issue needs further conversation as it's taking place in [LPP-52793](https://liferay.atlassian.net/browse/LPP-52793)

**Pull request closed without merging**